lib/Applify.pm

```diff
@@ -344,6 +375,28 @@ sub _sub {
   *$fqn = SUB_NAME_IS_AVAILABLE ? Sub::Name::subname($fqn, $code) : $code;
 }
 
+sub _subcommand_activate {
+  my ($self, $name) = @_;
+  return undef unless $name and $name =~ /^\w+/;
+  return undef unless $self->{subcommands}{$name};
+  $self->{subcommand} = $name;
+  {
+    no warnings 'redefine';
+    local *Applify::app = sub {
+      warn "Avoided deep recursion. Do not call app {} from subcommand block!\n";
+    };
+    $self->{subcommands}{$name}{code}->($self);
+  }
+  $self->{subcommands}{$name}{active} = 1;
```

Comment:
I don't think this is used anywhere else. (Remove it)

Reply:
Hmm, yeah I had thoughts of removing the `$self->{subcommand}` assignment and doing a `return map { $_->{name} } grep { $_->{active} } values %{$self->{subcommands}}` in `sub subcommand {}`

Reply:
I like your use of `$self->{subcommand}`